use-timescale/data-tiering/tiered-data-replicas-forks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Replicas and forks with tiered data
+excerpt: How tiered data works on replicas and forks
+product: [cloud]
+keywords: [tiered storage]
+tags: [storage, data management]
+---
+
+# How tiered data works on replicas and forks 
+
+There is one more thing that makes Tiered Storage even more amazing: when you keep data in the low-cost object storage tier,
+you pay for this data only once, independently of if you have a [high-availability replica][ha-replica]

```

Comment:
Not an English native-speaker :D, but it might sound better as: 
```suggestion
you pay for this data only once, regardless of whether you have a [high-availability replica][ha-replica]

```